pkg/cdi/container-edits.go

```diff
@@ -244,6 +274,9 @@ func (e *ContainerEdits) isEmpty() bool {
 	if e.IntelRdt != nil {
 		return false
 	}
+	if e.NetDevices != nil {
+		return false
+	}
```

Comment:
```suggestion
	if len(e.NetDevices) > 0 {
		return false
	}
```
This should match the logic for other slices.